src/models/predicate/link_v02.rs

```diff
@@ -23,23 +23,30 @@ pub struct LinkV02 {
     byproducts: ByProducts,
 }
 
-impl PredicateLayout for LinkV02 {
-    fn to_bytes(&self) -> Result<Vec<u8>> {
```

Comment:
`canonicalize` operation is performed by `Metablock`, can be removed and use standard Serializing.